examples/src/bin/composite_template.rs

```diff
@@ -0,0 +1,114 @@
+//! # Composite Template Example
+//!
+//! This sample demonstrates how to create a widget using GTK's composite templates.
+
+#[macro_use]
+extern crate glib;
+#[macro_use]
+extern crate gtk;
+
+use gio::{prelude::ApplicationExtManual, ApplicationExt};
+use glib::subclass::prelude::*;
+use glib::{Cast, StaticType};
+use gtk::WidgetExt;
+
+mod imp {
+    use super::*;
+    use glib::subclass;
+    use gtk::subclass::prelude::*;
+    use gtk::subclass::widget::*;
+    use gtk::WidgetExt;
+
+    /// The private struct, which can hold widgets and other data.
+    #[derive(Debug, CompositeTemplate)]
```

Comment:
```suggestion
    #[derive(Debug, gtk::CompositeTemplate)]
```

Reply:
I've changed this in a different way - mind taking a look?